Formula/fontforge.rb

```diff
@@ -47,6 +47,10 @@ def install
     args << "--without-libspiro" if build.without? "libspiro"
     args << "--without-libuninameslist" if build.without? "libuninameslist"
 
+    # Fix header includes
+    # fontforge/fontforge#3145
+    system "sed", "-i.bak", "s/#include \"fontforgevw.h\"/#include \"fontforgevw.h\"\\\n#include \"encoding.h\"/g", "fontforgeexe/startnoui.c"
```

Comment:
You'll want to use `inreplace` here, I suspect.

Reply:
Thanks.